API.md

```diff
@@ -13,10 +13,8 @@ that require loading a single resource. Once the handler is called, the applicat
 combine **bell** with the [**hapi-auth-cookie**](https://github.com/hapijs/hapi-auth-cookie) authentication scheme plugin.
 
 ```javascript
-var Hapi = require('hapi');
-var server = new Hapi.Server();
-
-server.connection({ port: 8000 });
+var { Server } = require('hapi');
```

Comment:
Might as well use `const`s and other ES6 features wherever possible now.

Reply:
Updated